fastlane/Fastfile

```diff
@@ -167,8 +167,10 @@ platform :ios do
           reject_build_waiting_for_review: false,
           submit_beta_review: true
         )
-      rescue => ex
-        UI.error("Something went wrong: #{ex}")
+      rescue Spaceship::UnexpectedResponse => ex
```

Comment:
have wee tested that fastlane still runs locally without throwing syntax errors or such? 👁️

Reply:
There doesn't seem to be any syntax errors, when run locally